### PR TITLE
[DOCFIX] Add additional YARN docs

### PR DESCRIPTION
**Pull request by aaudiber**
These docs describe how to run Alluxio with an existing YARN cluster, either standalone or as a YARN application

### CODE REVIEW

File: docs/en/Running-Alluxio-Yarn-Standalone.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+priority: 6
+---
+
+Alluxio should be run alongside YARN so that all YARN nodes have access to a local Alluxio worker.
+For YARN and Alluxio to play nicely together on the same nodes, it's important to inform YARN of
+the resources used by Alluxio. YARN needs to know how much memory and cpu to leave for Alluxio.
+
+## Allocating Resources for Alluxio
+
+#### Alluxio Worker Memory
```
Comment (calvinjia) [inline docs/en/Running-Alluxio-Yarn-Standalone.md]: Any reason for skipping `###`?

File: docs/en/Running-Alluxio-Yarn-Standalone.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+The Alluxio worker requires some memory for its JVM process and some memory for its ramdisk.
+1GB is generally fine for the JVM memory since this memory is only used for buffering and metadata. Data
+is stored on ramdisk. The amount of memory needed by the ramdisk is controlled by `alluxio.worker.memory.size`.
```
Comment (calvinjia) [inline docs/en/Running-Alluxio-Yarn-Standalone.md]: Mention that non-ram tiered storage is unrelated to YARN resource management?

File: docs/en/Running-Alluxio-Yarn-Standalone.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+`yarn-site.xml`. After determining how much memory to allocate to Alluxio on the node, subtract this from
+`yarn.nodemanager.resource.memory-mb` and update the parameter with the new value. The same should be done
+for `yarn.nodemanager.resource.cpu-vcores`.
+
+After updating the YARN configuration, restart YARN so that it picks up the changes:
+
+```bash
+$ ${HADOOP_HOME}/sbin/stop-yarn.sh
+$ ${HADOOP_HOME}/sbin/start-yarn.sh
+```
```
Comment (calvinjia) [inline docs/en/Running-Alluxio-Yarn-Standalone.md]: Perhaps add a conclusion stating how to run an application managed through yarn?

File: docs/en/Running-Alluxio-Yarn-Standalone.md (hunk 1 above)
Comment (aaudiber) [inline docs/en/Running-Alluxio-Yarn-Standalone.md]: changed to `###`

File: docs/en/Running-Alluxio-Yarn-Standalone.md (hunk 2 above)
Comment (aaudiber) [inline docs/en/Running-Alluxio-Yarn-Standalone.md]: done

File: docs/en/Running-Alluxio-Yarn-Standalone.md (hunk 3 above)
Comment (aaudiber) [inline docs/en/Running-Alluxio-Yarn-Standalone.md]: I'll soon be verifying YARN with RC2 anyway, so I'll develop a section then

**Pull request merged**